.circleci/config.yml

```diff
@@ -135,6 +136,9 @@ jobs:
       - run:
           name: Setup celo-monorepo
           command: |
+            set -e
+            mkdir ~/.ssh && ssh-keyscan github.com >> ~/.ssh/known_hosts
+            ssh-keygen -F github.com -l -f ~/.ssh/known_hosts | grep 'github.com RSA ${GITHUB_RSA_FINGERPRINT}' && echo "Github key verification successful"
```

Comment:
I think we need `set -e -o pipefail`
also single quotes prevent variable substitution.